src/test/java/com/hocs/test/glue/CreateCaseStepDefs.java

```diff
@@ -45,34 +46,28 @@ public void iAmPresentedWith(String userView) {
                 createCase.assertNoOptionsAvailable();
                 break;
             default:
-                fail(userView + " is not defined with CreateCaseStepDefs.iAmPresentedWith");
+                System.out.println(userView
+                        + " is not defined within " + getClass().getSimpleName()
+                        + " class, " + getMethodName() + " method");
+                userView = null;
+                assumeNotNull(userView);
         }
     }
 
-
-   /* public void iAmPresentedWith(String userView) {
-        switch (userView.toUpperCase()) {
-            case "NO CASE TYPES":
-                createCase.assertNoOptionsAvailable();
-                break;
-            default:
-                fail(userView + " is not defined with CreateCaseStepDefs.iAmPresentedWith");
-        }
-    }*/
-
-
-
     @Given("^I create a single case \"([^\"]*)\"$")
     public void createACaseTypeSpecificCase(String caseType) {
         switch (caseType.toUpperCase()) {
             case "DCU MIN":
                 createCase.createDCUMinSingleCase();
+                setSessionVariable("caseType").to(caseType);
```

Comment:
instead of having this written 3 times, if youre always going to be setting this session variable do it at the start of the method ie:
```
public void createACaseTypeSpecificCase(String caseType) {
        setSessionVariable("caseType").to(caseType);

        switch (caseType.toUpperCase()) {	       
            case "DCU MIN":	            
                createCase.createDCUMinSingleCase();	    
                break;
            case "DCU N10":
                createCase.createDC10SingleCase();
                break;
           case "DCU TRO":
                createCase.createDCTROSingleCase();
                break;
```